doc/examples/EEG_Example.md

```diff
@@ -0,0 +1,220 @@
+is an `xarray.Dataset` which contains all the feature information needed to train a machine
+learning model: feature values are stored as data variables, and the time series index/class
+label are stored as coordinates (a `channel` coordinate will also be used later for
+multi-channel data).
+
+### Custom feature functions
+Custom feature functions not built into `mltsp` may be passed in using the
+`custom_functions` keyword, either as a dictionary `{feature_name: function}`, or as a
+[dask graph](http://dask.pydata.org/en/latest/custom-graphs.html). Functions should take
+three arrays `times, measurements, errors` as inputs; details can be found in the
+`mltsp.featurize`
+[documentation](https://mltsp.readthedocs.org/en/latest/api/mltsp.featurize.html).
+Here we'll compute five standard features for EEG analysis provided by [Guo et al.
+(2012)](http://linkinghub.elsevier.com/retrieve/pii/S0957417411003253):
+
+```python
+import numpy as np
+import scipy.stats
+
+def mean_signal(t, m, e):
+    return np.mean(m)
+
+def std_signal(t, m, e):
+    return np.std(m)
+
+def mean_square_signal(t, m, e):
+    return np.mean(m ** 2)
+
+def abs_diffs_signal(t, m, e):
+    return np.sum(np.abs(np.diff(m)))
+
+def skew_signal(t, m, e):
+    return scipy.stats.skew(m)
+```
+
+Now we'll pass the desired feature functions as a dictionary via the `custom_functions`
+keyword argument.
+
+```python
+guo_features = {
+    'mean': mean_signal,
+    'std': std_signal,
+    'mean2': mean_square_signal,
+    'abs_diffs': abs_diffs_signal,
+    'skew': skew_signal
+}
+
+fset_guo = featurize.featurize_time_series(eeg["times"], eeg["measurements"], None,
+                                           list(guo_features.keys()), eeg["classes"],
+                                           custom_functions=guo_features, use_celery=True)
+print(fset_guo)
+```
+
+### Multi-channel time series
+The EEG time series considered here consist of univariate signal measurements along a
+uniform time grid. But
+[`featurize_time_series`](https://mltsp.readthedocs.org/en/latest/api/mltsp.featurize.html#mltsp.featurize.featurize_time_series)
+also accepts multi-channel data; to demonstrate this, we will decompose each signal into
+five frequency bands using a discrete wavelet transform as suggested by [Subasi
+(2005)](http://www.sciencedirect.com/science/article/pii/S0957417404001745), and then
+featurize each band separately using the five functions from above.
+
+```python
+import pywt
+
+n_channels = 5
+eeg["dwts"] = [pywt.wavedec(m, pywt.Wavelet('db1'), level=n_channels-1)
+               for m in eeg["measurements"]]
+fset_dwt = featurize.featurize_time_series(None, eeg["dwts"], None,
+                                           list(guo_features.keys()), eeg["classes"],
+                                           custom_functions=guo_features)
+print(fset_dwt)
+```
+
+The output featureset has the same form as before, except now the `channel` coordinate is
```

Comment:
It's present in the previous example too, it's just trivially `[0]`. It's mentioned in the previous paragraph that it's present but not yet used, so I think it makes sense to just refer to it without additional introduction here.